tests/integration_tests/cloud/test_cloud_workspaces.py

```
@@ -53,20 +44,31 @@
     )
     source.check()
 
-    cache = MotherDuckCache(
-        api_key=motherduck_api_key,
+    destination = DestinationMotherDuck(
```

Ruff (0.8.0)

Undefined name `DestinationMotherDuck`

(F821)